app/schemas/__init__.py

```
@@ -17,4 +17,8 @@
 
 __all__ = [
     # TODO: Add schema exports as they are implemented
+    "FarmerCreate",
```

Pylint (3.3.7)

[error] Undefined variable name 'FarmerCreate' in __all__

(E0603)